packages/cli/src/commands/access/add.ts

```diff
@@ -18,7 +18,7 @@ export default class AccessAdd extends Command {
   ]
 
   static args = {
-    email: Args.string({required: true}),
+    email: Args.string({required: true, description: 'Email address of the team member.'}),
```

Comment:
Unless there are multiple sentences, everything should be lowercase and have no period. So "email address of the team member"

Reply:
+1